static/app/views/performance/landing/widgets/widgets/slowScreens.tsx

```diff
@@ -0,0 +1,403 @@
+          extraQueryParams = {
+            ...extraQueryParams,
+            dataset: DiscoverDatasets.METRICS,
+          };
+
+          eventView.fields = [
+            {field: 'avg(measurements.time_to_initial_display)'},
+            {field: 'release'},
+          ];
+          const mutableSearch = new MutableSearch(eventView.query);
+          mutableSearch.addFilterValue('event.type', 'transaction');
+          mutableSearch.addFilterValue('transaction.op', 'ui.load');
+          eventView.query = appendReleaseFilters(
+            mutableSearch,
+            primaryRelease,
+            secondaryRelease
+          );
+          eventView.interval = getInterval(
+            pageFilter.selection.datetime,
+            STARFISH_CHART_INTERVAL_FIDELITY
+          );
+
+          return (
+            <EventsRequest
+              {...pick(provided, eventsRequestQueryProps)}
+              api={api}
+              yAxis={yAxis}
+              includePrevious={includePreviousParam}
+              includeTransformedData
+              partial={partialDataParam}
+              currentSeriesNames={currentSeriesNames}
+              field={eventView.getFields()}
+              query={eventView.getQueryWithAdditionalConditions()}
+              interval={interval}
+              hideError
+              onError={pageError.setPageError}
+              queryExtras={extraQueryParams}
+              topEvents={2}
+              referrer="performance-line-chart-widget"
+            />
+          );
+        },
+        transform: transformEventsChartRequest,
+      };
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [
+      props.chartSetting,
+      selectedListIndex,
+      mepSetting.memoizationKey,
+      primaryRelease,
+      secondaryRelease,
+    ]
+  );
+
+  const Queries = {
+    list: listQuery,
+    chart: chartQuery,
+  };
+
+  const assembleAccordionItems = provided =>
+    getItems(provided).map(item => ({header: item, content: getChart(provided)}));
+
+  const getChart = provided =>
+    function () {
+      const transformedReleaseSeries: {
+        [releaseVersion: string]: Series;
+      } = {};
+
+      const series = provided.widgetData.chart.data;
+      if (defined(series)) {
+        series.forEach(({seriesName: release, data}) => {
+          const isPrimary = release === primaryRelease;
+
+          const label = release;
+          const seriesData =
+            data.map(datum => {
+              return {
+                name: datum.name,
+                value: datum.value,
+              } as SeriesDataUnit;
+            }) ?? [];
+
+          const color = isPrimary ? CHART_PALETTE[3][0] : CHART_PALETTE[3][1];
+          transformedReleaseSeries[release] = {
+            seriesName: formatVersion(label),
+            color,
+            data: seriesData,
+          };
+        });
+      }
+
+      return (
+        <Chart
+          height={80}
```

Comment:
also, chart height should be the same as whatever the other chart height is
![Screenshot 2023-11-23 at 2 44 44 PM](https://github.com/getsentry/sentry/assets/63818634/1ede0750-d9c1-41c7-a70f-9e3ed16a73ca)